### PR TITLE
tikv: configuration change 6.0

### PR DESCRIPTION
**Pull request by you06**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by TomShawn — APPROVED** [review]
LGTM

**Comment by TomShawn** [comment]
@you06 Could you please involve a tech review? Thanks~

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -961,8 +961,8 @@ rocksdb 相关的配置项。
 
 ### `enable-pipelined-write`
 
-+ 开启 Pipelined Write 的开关。
-+ 默认值：true
++ 不再使用此参数来控制 Pipelined Write 的开关。开启时会使用原来的 Pipelined Write，关闭时会使用新的 Pipelined Commit 机制。
++ 默认值：false
```
Comment (you06) [inline tikv-configuration-file.md:965]: @Connor1996 PTAL

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1488,7 +1488,7 @@ Raft Engine 相关的配置项。
 ### `num-threads`
 
 + 处理备份的工作线程数量。
-+ 默认值：CPU * 0.5，但最大为 8 
++ 默认值：CPU * 0.5，但最大为 CPU 数量
```
Comment (you06) [inline tikv-configuration-file.md:1491]: @ethercflow PTAL

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -845,13 +845,13 @@ rocksdb 相关的配置项。
 ### `max-background-jobs`
 
 + RocksDB 后台线程个数。调整 RocksDB 线程池的大小时，请参考 [TiKV 线程池调优](/tune-tikv-thread-performance.md#tikv-线程池调优)。
-+ 默认值：8
++ 默认值：9
```
Comment (tabokie) [inline tikv-configuration-file.md:848]:
```suggestion
+ 默认值：7
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -845,13 +845,13 @@ rocksdb 相关的配置项。
...
 + 最小值：2
 
 ### `max-background-flushes`
 
 + RocksDB 用于刷写 memtable 的最大后台线程数量。
-+ 默认值：2
++ 默认值：3
```
Comment (tabokie) [inline tikv-configuration-file.md:854]:
```suggestion
+ 默认值：2
```

**Review by tabokie — COMMENTED** [review]
I'm assuming this default value was calculated under 8c deployment.

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -961,8 +961,8 @@ rocksdb 相关的配置项。
 
 ### `enable-pipelined-write`
 
-+ 开启 Pipelined Write 的开关。
-+ 默认值：true
++ 不再使用此参数来控制 Pipelined Write 的开关。开启时会使用原来的 Pipelined Write，关闭时会使用新的 Pipelined Commit 机制。
```
Comment (Connor1996) [inline tikv-configuration-file.md:964]:
```suggestion
+ 控制是否开启 Pipelined Write。开启时会使用的 Pipelined Write，关闭时会使用新的 Pipelined Commit 机制。
```

File: tikv-configuration-file.md (hunk 2 above)
Comment (ethercflow) [reply tikv-configuration-file.md:1491]: LGTM

Comment (ethercflow) [reply tikv-configuration-file.md:1491]: LGTM

**Review by ethercflow — APPROVED** [review]
LGTM

**Review by Connor1996 — APPROVED** [review]
LGTM

**Review by tabokie — APPROVED** [review]

File: tikv-configuration-file.md [hunk 6]
```diff
@@ -961,8 +961,8 @@ rocksdb 相关的配置项。
 
 ### `enable-pipelined-write`
 
-+ 开启 Pipelined Write 的开关。
-+ 默认值：true
++ 控制是否开启 Pipelined Write。开启时会使用的 Pipelined Write，关闭时会使用新的 Pipelined Commit 机制。
```
Comment (en-jin19) [inline tikv-configuration-file.md:964]:
```suggestion
+ 控制是否开启 Pipelined Write。开启时会使用旧的 Pipelined Write，关闭时会使用新的 Pipelined Commit 机制。
```

**Pull request merged**